test/functional/apps/dashboard/group3/panel_replacing.ts

```diff
@@ -51,7 +51,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.header.waitUntilLoadingHasFinished();
       const panelTitles = await PageObjects.dashboard.getPanelTitles();
       expect(panelTitles.length).to.be(2);
-      expect(panelTitles[0]).to.be(AREA_CHART_VIS_NAME);
+      expect(panelTitles.includes(AREA_CHART_VIS_NAME)).to.be(true);
```

Comment:
Is title order no longer guaranteed? Why was this change made? Maybe `dashboard.getPanelTitles` should have a comment documenting this?

Reply:
@ThomThomson I think the fact that I added sorting to the `layout` logic should have fixed this - can you confirm why you added this? Because I think it should be safe to remove - will test 👀

Reply:
Yes, I removed some sorting logic that meant that the order of these panel titles was no longer guaranteed so I had to update this test.

Hannah found that my removal of this logic was actually causing some edge cases so we decided to restore it. In doing that we should also restore this test to the way it used to be.

Reply:
Undid these changes in 09d366f839def634d5fd0280a844bf996760789f - I had go back to the `panelsInOrder` idea to make the layout consistent when shrinking + to make the HTML line up with the expected order, but this test works as expected now :+1: